tools/dts.js

```diff
@@ -1,323 +1,323 @@
-'use strict';
```

Comment:
looks like something made a bunch of whitespace changes here, will revert so the diff is readable.